content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/amp.html

```diff
@@ -0,0 +1,50 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<section class="cmp-search" role="search"
```

Comment:
@gaetanmarmasse - I wasn't able to get the search component to display results on an AMP page when confirming an input. Following console message is displayed:
<img width="550" alt="Screenshot 2020-01-23 at 17 20 18" src="https://user-images.githubusercontent.com/25616660/73012728-373aed00-3e17-11ea-8337-6e2a71c31933.png">

Reply:
@richardhand , This is happening if you try to add the search component in the editable template without specifying the `searchTermMinimumLength="1"`.

`amp-autocomplete`, requires a JSON with actual data in it. We found it best to not modify the `searchresults` servlet provided with core components and call it with the "*" fulltext search. 
This will work only if there is an associated content policy that sets the `searchTermMinimumLength` to 1. Default is 3 which will result in an empty JSON.